docs/plugins/rcpt_to.routes.md

```diff
@@ -1,4 +1,4 @@
-# `rcpt_to.routes`
+# rcpt_to.routes
```

Comment:
Maybe? I think so.

On Thu, Aug 18, 2016 at 11:35 AM, Matt Simerson <notifications@github.com>
wrote:

> In docs/plugins/rcpt_to.routes.md
> <https://github.com/haraka/Haraka/pull/1568#discussion_r75331888>:
>
> > @@ -1,4 +1,4 @@
> > -# `rcpt_to.routes`
> > +# rcpt_to.routes
>
> @baudehlo <https://github.com/baudehlo>, will this work now with the new
> html page rendering?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/haraka/Haraka/pull/1568/files/ec5b41f392d310c8df0401503eafc74663754164#r75331888>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAobYw_YWyyYDVEdkwLZ62SHQTK6FU4Aks5qhHvAgaJpZM4Jmyno>
> .
>